.gitignore

```diff
@@ -24,6 +24,10 @@ junit*
 dist/
 build/
 
+# Node.js
+node_modules
+npm-debug.log
+
```

Comment:
Not sure why this diff is shown. Isn't this already in main?
Perhaps a different line ending? (CRLF)